--- manager.c.orig	2011-06-25 12:29:49.449486778 +0300
+++ manager.c	2011-06-25 12:29:55.730142667 +0300
@@ -5541,7 +5541,7 @@ static int generic_http_callback(struct
 		"Content-type: text/%s\r\n"
 		"Cache-Control: no-cache;\r\n"
 		"Set-Cookie: mansession_id=\"%08x\"; Version=1; Max-Age=%d\r\n"
-		"Pragma: SuppressEvents\r\n",
+		"Pragma: SuppressEvents",
 		contenttype[format],
 		session->managerid, httptimeout);
 
